cmd/apg/admin_service.go

```diff
@@ -18,7 +18,8 @@ var AdminConfig *viper.Viper
 var AdminClient *gapic.AdminClient
 var AdminSubCommands []string = []string{
 	"get-status",
-	"list-projects",
+	"migrate-database",
+	"poll-migrate-database", "list-projects",
```

Comment:
Minor, should `list-projects` on a separate line?

Reply:
I would think so, but this is auto-generated and automatically-formatted code.